docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -24,54 +24,50 @@
 
 @endsphinxdirective
 
-## Introduction
-
 Model Optimizer is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
 
-Using Model Optimizer tool assumes you already have a deep learning model trained using one of the supported frameworks: TensorFlow, PyTorch, PaddlePaddle, MXNet, Caffe, Kaldi, or represented in ONNX* format. Model Optimizer produces an Intermediate Representation (IR) of the model, which can be inferred with [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
+When using Model Optimizer, you are assumed to already have a deep learning model that is trained using one of the supported frameworks: TensorFlow, PyTorch, PaddlePaddle, MXNet, Caffe, Kaldi, or represented in ONNX format. Model Optimizer produces an Intermediate Representation (IR) of the model, which can be inferred with [OpenVINO™ Runtime](../OV_Runtime_UG/openvino_intro.md).
 
-> **NOTE**: Model Optimizer does not infer models. Model Optimizer is an offline tool that converts a model into IR and optimizes before the inference takes place.
+> **NOTE**: Model Optimizer does not infer models. It is an offline tool that converts a model into IR and optimizes it before the inference takes place.
```

Comment:
How about we remove the disclaimer? Rewording the previous paragraph should be enough to drive the message and we can avoid repeating the same thing twice.

Reply:
You are right, no need for repetition.

Reply:
I think this disclaimer is not repetitive information. The previous paragraph says what MO does, while this disclaimer clearly says what it does not do to avoid confusion. I suggest we keep it.

Reply:
I agree with Yuan. I saw Sebastian rewording and I don't know if that is only me but it is really unfitting that when we describe item for the first time in article we start with stuff it can't do instead of what it is suppose to do. Also by leaving it as disclaimer we highlight it to reader to pay more focus on information what MO shouldn't be used for.

Reply:
What I meant is that we have two fragments. One states what  MO does and one stated what it does not do. It is like saying:
Ov converts to IR, which you can later infer with OpenVINO™ Runtime.
Remember that MO does not infer models, use OpenVINO™ Runtime for that.

I think it does count as a repetition :) When I mentioned rewording the previous paragraph I meant the rewording already in place, not implementing the disclaimer but in other words :) This way we actually omit talking about anything MO can't do. Not one negation in the previous paragraph :D Let's take a second look at the entire fragment.

Reply:
I think there was a reason to highlight that in a Note style, could be coming from a customer because we didn't say things explicitly. Can we keep that?

Reply:
I think in the current version it is quite clear, but if you strongly feel like the disclaimer should stay, let's leave it and see how it looks :) Let's just shorten it to the "I don't infer!" part, to avoid repetition.

Reply:
```suggestion
> **NOTE**: Model Optimizer does not infer models. 
```
And so here it goes ;)